pkg/system/phase2_creating.go

```diff
@@ -889,6 +890,20 @@ func (r *Reconciler) UpgradeMigrateDB() error {
 			return err
 		}
 		if phase == "" {
+			// when starting - restart the db pod. This is a fix for https://bugzilla.redhat.com/show_bug.cgi?id=1922113
```

Comment:
Why not inside of the Prepare it's part of that phase  no? I don't think we should do it here